src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        var supplierFigure = new FigureSupplierImpl();
+        var supplierColor = new ColorSupplier();
```

Comment:
It's recommended not to use 'var' for variable declarations in this context. Instead, specify the type explicitly for better readability and understanding of the code.